js/views/startScreen.js

```diff
@@ -25,6 +25,10 @@ var StartScreen = React.createClass({
     this.handleResize();
   },
 
+  componentWillReceiveProps: function() {
```

Comment:
Gotcha. Let's add a param to `handleResize()` to update state with nextProps. Will this work?
```
componentWillReceiveProps: function(nextProps) {
    if (nextProps.contentTree.description != this.props.contentTree.description) {
      this.handleResize(nextProps.contentTree.description);
    }
  },

  handleResize: function(text) {
    var description = text ? text : this.props.contentTree.description;
    if (ReactDOM.findDOMNode(this.refs.description)){
      this.setState({
        descriptionText: Utils.truncateTextToWidth(ReactDOM.findDOMNode(this.refs.description), description)
      });
    }
  },
```

We should always check for prop changes because this method gets called a lot, even if props have not changed. See https://facebook.github.io/react/docs/react-component.html#componentwillreceiveprops